services/tenant-ui/frontend/src/components/issuance/credentialtemplate/CreateCredentialTemplateForm.vue

```diff
@@ -0,0 +1,160 @@
+<template>
+    <h3 class="mt-0 mb-5">Create Credential Template</h3>
+    <div class="form-demo">
```

Comment:
Doesn't look like the 'form-demo' class is applying anything anywhere?
This could be a good place (or on the form element itself) to apply a global 'tenant-ui-form' class that we apply global style to all sub elements to make form spacing what we want, have some thoughts about that for the near future)